src/qt/locale/umbra_af_ZA.ts

```diff
@@ -659,7 +659,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <name>PaymentServer</name>
     <message>
         <location filename="../paymentserver.cpp" line="+107"/>
-        <source>Cannot start spectre: click-to-pay handler</source>
+        <source>Cannot start spectrecoin: click-to-pay handler</source>
```

Comment:
This line of text occours 14 times. Shouldn't it be uppercase as on the other ones?